### PR TITLE
MES-3077 - added duration metrics

### PR DESCRIPTION
**Pull request by chrisnappin**
# Description and relevant Jira numbers
Converted to use common logging, and added new duration metrics for SQL and Dynamo
Fixed security issues with dependencies

## Pull Request checklist

- [*] [WIP] tag removed from PR title
- [*] PR has an understandable description

## Git feature branch checklist

- [ ] branch name comply with our branching strategy
- [ ] git branch contains relevant JIRA ticket number
- [*] branch rebased against the latest develop

## Sign off process checklist

- [*] Code has been tested manually
- [ ] Tested by QA
- [ ] PO's approval
- [ ] Reviewers selected in Github
- [ ] PR link added to JIRA

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,10 @@
     "url": "git+https://github.com/dvsa/mes-data-replication-service.git"
   },
   "dependencies": {
+    "@dvsa/mes-journal-schema": "^1.0.8",
```
Comment (ghost) [inline package.json:13]: Why have journal schema and types packages moved from `devDependencies`, aren't they only used at compile time?

File: tslint.json [hunk 2]
```diff
@@ -2,9 +2,13 @@
...
     "no-duplicate-variable": true,
     "no-unused-variable": false,
+    "no-console": [true],
     "quotemark": [true, "single"],
     "indent": [true, "spaces", 2],
     "max-line-length": [true, 120]
   },
-  "extends": "tslint-config-airbnb"
+  "extends": "tslint-config-airbnb",
+  "linterOptions": {
```
Comment (ghost) [inline tslint.json:11]: Is disabling the linter intentional?

**Review by ghost — COMMENTED** [review]
Looks good, couple of comments

File: package.json (hunk 1 above)
Comment (chrisnappin) [reply package.json:13]: Yes, they are used at runtime

File: tslint.json (hunk 2 above)
Comment (chrisnappin) [reply tslint.json:11]: Disabling the console.log check for the non-prod code

Comment (ghost) [reply tslint.json:11]: But `*.ts` is the prod code? Or am I misunderstanding?

File: package.json (hunk 1 above)
Comment (ghost) [reply package.json:13]: I didn't know this, can you elaborate on how please?

**Review by rstepnowski — APPROVED** [review]

**Review by jameshighfield — APPROVED** [review]

**Pull request merged**